src/app/services/postman-sms/postman-sms.service.ts

```diff
@@ -156,6 +223,137 @@ class PostmanSmsService {
       )
     })
   }
+
+  public sendBouncedSubmissionSms(
+    recipient: string,
+    recipientEmail: string,
+    adminId: string,
+    adminEmail: string,
+    formId: string,
+    formTitle: string,
+  ): ResultAsync<true, SmsSendError | InvalidNumberError> {
+    const logMeta = {
+      action: 'sendBouncedSubmissionSms',
+      formId,
+    }
+
+    logger.info({
```

Comment:
nit: is this useful to log as info? Seems like it will add a lot of logs that we might not require

Reply:
Good catch. Log is already sent by their respective caller, and this adds very little value.

1dbc994ff